Tools/CISupport/ews-build/steps.py

```diff
@@ -5878,10 +5878,9 @@ def convert_build_to_os_name(self, build):
             return 'Unknown'
 
         build_to_name_mapping = {
+            '15': 'Sequoua',
```

Comment:
Sequoua -> Sequoia :O

Reply:
Yep, discovered this via the unit tests 💯